### ISSUE TITLE
Type error with setQueryData when blitz generates the [edit].tsx file on 0.25.0-canary.1

### ISSUE THREAD

**Issue by abuuzayr** [blitz-js/blitz#1398]
### What is the problem?

`Property 'setQueryData' does not exist on type 'Pick<QueryResult<Profile, unknown>, "canFetchMore" | "clear" | "error" | "failureCount" | "fetchMore" | "isError" | "isFetched" | "isFetchedAfterMount" | "isFetching" | ... 10 more ... | "updatedAt"> & QueryCacheFunctions<...>'.ts(2339)`

### Steps to Reproduce

1. `blitz new test-app`
2. `cd test-app`
3. `blitz db migrate`
4. `blitz generate all something`
5. `blitz db migrate`
6. `git add . && git commit -m 'Add something model'` <-- or test the husky pre-commit hook / run its commands

Error:

```bash
app/somethings/pages/somethings/[somethingId]/edit.tsx:11:20 - error TS2339: Property 'setQueryData' does not exist on type 'Pick<QueryResult<Something, unknown>, "error" | "updatedAt" | "canFetchMore" | "clear" | "failureCount" | "fetchMore" | "isError" | "isFetched" | "isFetchedAfterMount" | ... 10 more ... | "status"> & QueryCacheFunctions<...>'.

11   const [something, {setQueryData}] = useQuery(getSomething, {where: {id: somethingId}})
```

### Versions

```
macOS Catalina | darwin-x64 | Node: v14.15.0

blitz: 0.25.0-canary.1 (global)
blitz: 0.24.3 (local)

  Package manager: yarn
  System:
    OS: macOS 10.15.6
    CPU: (4) x64 Intel(R) Core(TM) i5-4260U CPU @ 1.40GHz
    Memory: 16.58 MB / 8.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 14.15.0 - /usr/local/bin/node
    Yarn: 1.22.5 - ~/.yarn/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/cli: 2.9.0 => 2.9.0
    @prisma/client: 2.9.0 => 2.9.0
    blitz: 0.24.3 => 0.24.3
    react: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8
    react-dom: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8
    typescript: 4.0.3 => 4.0.3
```

### Other

Please include applicable logs and screenshots that show your problem.

**Comment by flybayer** [comment]
@abuuzayr looks like this is because the version of blitz installed in the app is `0.24.3` instead of `0.25.0-canary.1` 🤔

**Comment by flybayer** [comment]
Should be fixed with `0.25.0`
